### DISCUSSION TITLE
Coplit-agent-win.exe is infected with Win64:Evo-gen [Susp], flagged by Avast Antivirus

### DISCUSSION THREAD

**Discussion by Fauzdar1** [community/community#20323] [Copilot Conversations]
I've installed Github Copilot in Android Studio and this morning the moment I opened Android Studio, Avast showed a virus alert stating that the `copilot-agent-win.exe` is infected with `Win64:Evo-gen [susp]` and has been moved to quarantine, see the screenshot attached. 

Is this a false positive or should I do a clean install of Github Copilot? 

![image](https://user-images.githubusercontent.com/62703211/178239936-ffbe23ff-d4f1-44a8-bfc0-0159a925463b.png)

**Comment by bbtrash** [comment]
Hello, similar problems: [20268](https://github.com/orgs/github-community/discussions/20268) and [20247](https://github.com/orgs/github-community/discussions/20247)

**Reply by Fauzdar1** [reply]
None of which lists any solution and both are new and don't appear in search results.

**Reply by chegrane** [reply]
McAfee : I have the same problem here, detected by McAfee.  I have already copilot installed on my machine, and it's work fine. I want to update it on (July 12, 2022), So McAfee say no, and delete it (\copilot-agent-win.exe detected as Trojan).

**Comment by naXa777** [comment]
I used this extension without problems till now.

Today copilot-agent-win.exe is detected as virus by BitDefender and moved to quarantine
![image](https://user-images.githubusercontent.com/3858548/184370511-c1acd90f-3f1d-4e93-bb28-8cc61d7f969b.png)
GitHub copilot doesn't work without this file.

I've also double-checked the file by VirusTotal scanner (online). result:
![image](https://user-images.githubusercontent.com/3858548/184370949-9ce47019-87bd-46d6-b3b3-be3f6580daf6.png)

https://www.virustotal.com/gui/file/56d7e3b2ef797f577fcd69c4443464a9bd087d2d0b1507713570df1bc4399f5f
